### DISCUSSION TITLE
Make a notch to 3d semi-cylinder sample

### DISCUSSION THREAD

**Discussion by rashidcr8585** [neperfepx/neper#144] [Q&A] [answered]
Hello there, I want to make a notch to my 3d semi-cylinder sample in neper, is there anyone to help me for solving this issue?

**Comment by rquey** [comment, marked as answer]
See https://github.com/rquey/neper/discussions/107 for a complete discussion on this.

**Reply by kmmm70** [reply]
Rashidcr8585,

You should follows Romain's suggestion on post #107. I followed his suggestions and now can create 3D simple example as shown. Thanks Romain.
![img](https://user-images.githubusercontent.com/57537961/119600007-4cab9700-be10-11eb-893a-8483249df78c.png)

**Comment by rashidcr8585** [comment]
Hi Rquey/Neper thank you for your reply, but I don't know how to creat
notch in 3d sample like SCB Specimens
